lib/datadog/appsec/processor.rb

```diff
@@ -60,6 +62,18 @@ def extract_schema?
           Datadog.configuration.appsec.api_security.enabled &&
             Datadog.configuration.appsec.api_security.sample_rate.sample?
         end
+
+        def remove_empty_entries(entries)
+          entries.each_with_object({}) do |(k, v), acc|
+            acc[k] = v unless empty_value?(v)
+          end
+        end
+
+        def empty_value?(v)
+          return true unless v
```

Comment:
That would return `true` for `v == false`

Reply:
Absolutely true 😄